semantic-conventions/README.md

```diff
@@ -44,8 +57,16 @@ const span = tracer.startSpan(spanName, spanOptions)
 
 ### Unstable SemConv
 
+```bash
+npm install --save-exact @opentelemetry/semantic-conventions
+```
+
+**Note**: Because the "incubating" entry-point may include breaking changes in
+minor versions, it is recommended that users of this entry-point depend on a
+pinned (exact) version of the package.
```

Comment:
```suggestion
**Note**: Because the "incubating" entry-point may include breaking changes in
minor versions, it is recommended that users of this entry-point depend on a
pinned (exact) version of the package or temporarily duplicate hardcoded values.
```

Java is currently hardcoding non-stable values in instrumentations, and I believe that was a recommendation from the semconv group somewhere as well (lost the link but I've been meaning to open a PR there as well since I heard it)

Reply:
Found the link and mentioned in a separate comment [Semantic Conventions guidelines](https://opentelemetry.io/docs/specs/semconv/non-normative/code-generation/#semantic-conventions-artifact-structure).

Reply:
handled in commit 8fedc235e